packages/docs/docs/releases.md

```
@@ -1,5 +1,18 @@
 # Release Notes
 
+## 26.2.1
+
+Release date: 2026-02-22
+
+This release contains two critical security fixes for users of SimpleFin, Pluggy or the multi-user feature. If you do not use these features: feel free to skip this version.
+
+**Docker Tag: v26.2.1**
+
+#### Bugfixes
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)